doc/devdocs/guidance.md

```diff
@@ -43,27 +46,32 @@ Use [XML resource file(.resx)][Resx Files] to store the UI display strings and [
   </data>
 ```
 
-- Use [`Resource Manager`][Resource Manager] to consume strings in code.
+- Use [Resource Manager][Resource Manager] to consume strings in code.
+
 ```csharp
 System.Resources.ResourceManager manager = new System.Resources.ResourceManager(baseName, assembly);
 string validUIDisplayString = manager.GetString("ValidUIDisplayString", resourceCulture);
 ```
 
-In case of Visual Studio is used to create the resource file. Simply use the `Resources` class in auto-generated `Resources.Designer.cs` file to access the strings which encapsulate the [`Resource Manager`][Resource Manager] logic.

```

Comment:
According to https://docs.microsoft.com/en-us/contribute/text-formatting-guidelines#code-style we should stick to the former inline code formatting class, method, ...,nuget package names or registry keys. The only exception I'm aware of is if a code element (e.g. class name) is a link at the same time. In that case, no (inline) code formatting should be used.

Reply:
Hm, I should double-check then. Must have had some other guideline in mind, maybe thinking of MS docs? Or maybe I was a bit too enthousiastic 😆

Reply:
When looking at the page from your link, you would be totally right. That page does say:

>Why? Older style guides specify bold for many of these text elements. However, most articles are localized, and code style tells the translator to leave that part of the text untranslated.

After Clint said "MS docs and Dev docs are two different beasts", I found https://docs.microsoft.com/en-us/style-guide/developer-content/formatting-developer-text-elements which is much more targeted for Dev environment/people. And there we see very few `code` styling. Furthermore, (afaik.) "our" docs would not get translated, so that eleminates _that_ risk/reason.

Reply:
Interesting. I just asked for clarification (https://github.com/MicrosoftDocs/microsoft-style-guide/issues/315). I still prefer (inline) code style for code related things. It is fairly common and even Microsoft uses this for the dev documentation. I think it would be great if all developer documentation would follow the same standard. Since the "old" Standard using bold text does not support the localization use case we should also adopt the new standard. It shouldn't make any difference if we plan to localize it or not.

Reply:
> It is fairly common and even Microsoft uses this for the dev documentation

Docs have been written for years and guidelines change(d) in the mean time. Quote from Matt:
>The expectation has been to keep the older, existing docs readable/usable, but not necessarily going back to bring everything up to date with the current (and continually evolving) rules and guidelines. Occasionally there will be an effort to go back and clean up something essential, like links, if needed... but an effort to completely bring everything up to the current guidance may be a bit ambitious.

https://github.com/MicrosoftDocs/windows-uwp/issues/3837#issuecomment-1130508681 - this full issue might be of interest to you too.

I believe the difference between scopes - "public" MS docs versus Dev guidelines - is significant and is valid in this case. But I am not working for Microsoft and I am not making the rules :)

Reply:
@CleanCodeDeveloper the style-guide repo is totally dead, no point in waiting for answer. What now?

Reply:
We have two rivaling style guides: 

**Style Guide 1**: https://docs.microsoft.com/en-us/style-guide/developer-content/formatting-developer-text-elements
**Style guide 2**: https://docs.microsoft.com/en-us/contribute/text-formatting-guidelines#code-style.

I personally prefer **Style guide 2** because: 
1. **Style guide 2** looks like a successor for **Style guide 1**
2. **Style guide 2** supports localization
   2a. I expect that **Style guide 2** will win in the long run because it is more universal
3. **Style guide 2** seems to be more common (highly subjective!)
    3a. Case in point: You would need less changes to migrate our developer docs to **Style guide 2** than you need for **Style guide 1**

Anyway, I will leave the decision up to you. It is more important to have a style guide and apply it consequently than the choice of the actual style guide.

Reply:
Yeah, it's a complicated thing, also when you see that old MS docs itself are not actively updated to new styles. For now, I do think that style 1 is more targeted at _dev_ docs, so if you're OK with it, I'll just go with this now.